doc/manuals/orion-api.md

```diff
@@ -1473,6 +1474,16 @@ The particular alteration type can be got in notifications using the
 This section contains the topics that, due to implementation decision, the behaviour is differs from the described in the 
 original NGSIv2 specification.
 
+## `actionType` metadata
+
+From original NGSIv2 specification section "Builtin metadata", regarding `actionType` metadata:
+
+> Its value depend on the request operation type: `update` for updates,
+> `append` for creation and `delete` for deletion. Its type is always `Text`.
+
+Current Orion implementation supports `update` and `append`. The `delete` case will be
+supported upon completion of [this issue](https://github.com/telefonicaid/fiware-orion/issues/1494).
```

Comment:
It seems it is already completed

Reply:
I think was a wrong number. See further comment.

NTC